src/components/Book.jsx

```diff
@@ -15,21 +15,48 @@ function Book(props) {
   };
 
   return (
-    <div>
-      <ul>
-        <li>
-          {title}
-          {' '}
-          by
-          {author}
-          {' '}
-          {category}
-&nbsp;
-          <button onClick={() => handleDel()} type="button">
-            Delete
+    <div className="book">
+      <div className="p-1">
+        <div className="titles">
+          <h4 className="category">{category}</h4>
+          <h2 className="title">{title}</h2>
+          <p className="author">{author}</p>
+        </div>
+        <div className="buttons">
+          <button className="btn comment-btn" type="submit">
+            Comment
           </button>
-        </li>
-      </ul>
+          <button
+            onClick={() => handleDel()}
+            type="button"
+            className="btn remove-btn"
+          >
+            Remove
+          </button>
+          <button className="btn edit-btn" type="submit">
+            Edit
+          </button>
+        </div>
+      </div>
+      <div className="p-2">
+        <div className="circle">
+          <svg>
+            <circle cx="20" cy="20" r="20" />
+            <circle cx="20" cy="20" r="20" />
+          </svg>
+        </div>
+        <div className="infos">
+          <h3>8%</h3>
+          <span>completed</span>
+        </div>
```

Comment:
- [ ] As I mentioned above :point_up: please stick to the [Zaplin](https://zeplin.io/) design but your `font-style` and `circle` is not the same as [Zaplin](https://zeplin.io/) design so kindly fix these issues :+1: check below :point_down:  the comparison between your design and [Zaplin](https://zeplin.io/) design :slightly_smiling_face:

**Your Design**
![Screenshot from 2023-06-23 16-12-52](https://github.com/husush23/Book-Store/assets/45226848/70003fde-844b-4969-9487-2e8709860e2c)

**Zaplin Design** 
![Screenshot from 2023-06-23 16-12-46](https://github.com/husush23/Book-Store/assets/45226848/3a066cb0-3e88-42c8-9d86-958e5dccba78)